### ISSUE TITLE
1.3.0 python task  bug 

### ISSUE THREAD

**Issue by feiyalun** [apache/incubator-dolphinscheduler#2769]
![图片](https://user-images.githubusercontent.com/37201886/82434747-e8038100-9ac5-11ea-866f-ee692542cc2c.png)

import os
return_code=os.system("python /data/test/shell/ds_demo/m01_etl/dwd/dwd_m01_daily_order_classify.py AA_${day}.dir")
print 'return_code=',return_code
if return_code!=0: exit(1)#异常退出
![图片](https://user-images.githubusercontent.com/37201886/82434851-0e292100-9ac6-11ea-86eb-8ea1abe1cd15.png)
居然异常退出， 返回码是0.
![图片](https://user-images.githubusercontent.com/37201886/82434792-f94c8d80-9ac5-11ea-9607-80edce6fd184.png)
我自己的脚本是：
![图片](https://user-images.githubusercontent.com/37201886/82434989-3ca6fc00-9ac6-11ea-83b4-08926367ace9.png)
谁能给解释一下，为啥给判断失败，我返回码是0 ，我之前测过，返回值是Int类型。为啥给我失败退出

**Comment by feiyalun** [comment]
希望有人能给解释一下 为啥会判断失败，导致重试一次。

**Comment by feiyalun** [comment]
![图片](https://user-images.githubusercontent.com/37201886/82435430-e2f30180-9ac6-11ea-8389-5a6671679316.png)

**Comment by lgcareer** [comment]
@feiyalun Hi,Whether the question you said above resolved,I tested on the 1.3.2 branch and it is successful.
The python scirpt is the following.
```
import os
return_code=os.system('cd /usr/local')
print 'return_code=',return_code
if return_code!=0: exit(1)#异常退出
```

**Comment by dailidong** [comment]
no update message,  I will close this issue temporarily
